subjects/devops/orchestrator/audit/README.md

```diff
@@ -0,0 +1,232 @@
+#### General
+
+##### Check the Repo content
+
+A `README.md` and `orchestrator.sh` and `Vagrantfile` files and all files used to create and delete and manage the student infrastructure must be submitted in the repo.
+
+```console
+.
+├── Manifests
+│   └── [...]
+├── Scripts
+│   └── [...]
+├── Dockerfiles
+│   └── [...]
+└── Vagrantfile
+```
+
+###### Are all the required files present?
+
+##### Ask the following questions to the group or student
+
+- What are Container orchestration and what are its benefits?
+
+- What is Kubernetes and what is its main role of it?
+
+- What are K3s and what is their main role?
+
+###### Did the student reply correctly to the questions?
+
+##### Check the Student Documentation
+
+The Documentation must exist in the `README.md` file!
+
+###### Did the README.md file contains the all required information about the solution(Prerequisites, Configuration, Setup, Usage, ...)?
+
+##### Check the docker images in the docker hub:
+
+###### Do the used docker images in the YAML manifest uploaded in the student docker hub account?
```

Comment:
`Are the docker images used in the YAML manifest uploaded from the student's Docker Hub account?`

Reply:
fixed